evidence/model/evidence_metadata.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+type ResponseSearchEvidence struct {
+	Data EvidenceData `json:"data"`
+}
+
+type EvidenceData struct {
+	Evidence Evidence `json:"evidence"`
+}
+
+type Evidence struct {
+	SearchEvidence SearchEvidence `json:"searchEvidence"`
+}
+
+type SearchEvidence struct {
+	Edges []SearchEvidenceEdge `json:"edges"`
+}
+
+type SearchEvidenceEdge struct {
+	Node EvidenceMetadata `json:"node"`
+}
+
+type EvidenceMetadata struct {
```

Comment:
It looks like something we might want to use in future implementations don't we want it to match the current code evidence struct:
https://git.jfrog.info/projects/JFROG/repos/evidence/browse/service/internal/model/model_evidence_entity.go#5